Automattic/jetpack
# Map Block: Use the block styles API for choosing map themes

This PR switches the map block over to using the block styles attribute, rather than the custom theme picker.

Fixes #14559

<img width="1120" alt="" src="https://user-images.githubusercontent.com/352291/75655352-c6101480-5cb5-11ea-92c1-d1ccdbf62e64.png">

#### Changes proposed in this Pull Request:
* See discussion on #14559 and linked issues.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Change to existing feature.

#### Testing instructions:

* Add a new map block, test that the block style switches the map theme as expected.
* Open an post with an old map block, check that in correctly converts to the new style.

#### Proposed changelog entry for your changes:
* Map Block: Improve the map theme picker.
